src/Sarif/GitHelper.cs

```diff
@@ -87,7 +87,7 @@ public string GetCurrentCommit(string repoPath)
         public string GetBlame(string filePath)
         {
             return GetSimpleGitCommandOutput(
-                Path.GetDirectoryName(filePath),
+                GetTopLevel(filePath),
                 args: $"blame -f --porcelain {Path.GetFileName(filePath)}",
```

Comment:
This line is also required for this change to work, because `filePath` includes the intervening nested directory path between the repo root and the file.
```suggestion
                args: $"blame -f --porcelain {filePath}",
```

Reply:
ok, added a simple unit test. we need to go back and do more exhaustive testing soon.